packages/babel-plugin-bugfix-v8-spread-parameters-in-optional-chaining/test/fixtures/assumption-noDocumentAll/basic/input.mjs

```diff
@@ -0,0 +1,7 @@
+fn?.(...b, 1);
+
+a?.b(...c, 1);
+
+a?.b?.(...c, 1);
```

Comment:
As an optimization, it would be enough to transform this to
```js
(_a2$b = (_a = a)?.b) === null || _a2$b === void 0 ? void 0 : _a2$b.call(_a2, ...c, 1);
```
instead of
```js
(_a2 = a) === null || _a2 === void 0 ? void 0 : (_a2$b = _a2.b) === null || _a2$b === void 0 ? void 0 : _a2$b.call(_a2, ...c, 1);
```

Reply:
Yeah I mentioned that this PR is sub-optimal for nested chains. But because it's an edge case and we already have to maintain two optional chain transform, I prefer to leave it as-is.